Sources/LiveViewNative/BuiltinRegistry.swift

```diff
@@ -7,7 +7,13 @@
 
 import SwiftUI
 
-struct BuiltinRegistry {
+protocol BuiltinRegistryProtocol {
```

Comment:
Am I right in my understanding that this protocol is necessary to tie the typealias to the opaque return type of the `decodeModifier` implementation, so that there's something that can be explicitly spelled out in the `ModifierContainer`?

If so, I'd add a brief comment to that effect

Reply:
Yep, exactly. I'll add a comment.